### ISSUE TITLE
Hyperparameters Setting

### ISSUE THREAD

**Issue by BinWang28** [yaohungt/Multimodal-Transformer#5]
Hi author,

Many thanks for sharing the code. I am trying to obtain the result from the paper for CMU-MOSEI dataset. I am using the current released version and changed the Hyperparameters setting for CMU-MOSEI dataset according to the appendix. (refer: Table 5 in your paper.)

But currently I haven't got similar results (About 1-2% lower for accuracy.) Is there anything else need to be modified for me to obtain similar results? Appreciate for any help.

Have tried for both aligned and unaligned. Haven't tried the other two datasets yet.

**Comment by yaohungt** [comment]
CMU-MOSEI is a very small dataset which is easy to overfit. You may want to try more random seeds or do some regularization/ early stopping for preventing overfitting.

**Comment by shaojieb** [comment]
Just to add to @yaohungt's comment above--- I just tried again and is able to reproduce most of the numbers for CMU-MOSEI in one run. I would suggest slightly tuning the dropout rates (especially the embedding dropout and residual dropout) and the number of levels of the transformers, while re-running a few times with different seeds (you may sometimes get results better than in the table).

**Comment by BinWang28** [comment]
Many thanks for the reply. I will try a few more time and see how it works.

**Comment by sylvia5monthes** [comment]
@BinWang28 were you able to get the results? I'm having a similar issue.

**Comment by JKBox** [comment]
@jerrybai1995 could you kindly share the hyperparameters in MOSEI aligned dataset?
